Sources/Examples/RouteGuide/Client/RouteGuideClient.swift

```diff
@@ -23,7 +23,7 @@ import RouteGuideModel
 
 /// Loads the features from `route_guide_db.json`, assumed to be in the directory above this file.
 func loadFeatures() throws -> [Routeguide_Feature] {
-  let url = URL(fileURLWithPath: #file)
+  let url = URL(fileURLWithPath: #fileID)
```

Comment:
This one should be `#filePath` as we're finding a file relative to this source file.

```suggestion
  let url = URL(fileURLWithPath: #filePath)
```